lottie/src/main/java/com/airbnb/lottie/network/LottieNetworkCacheProvider.java

```diff
@@ -0,0 +1,18 @@
+package com.airbnb.lottie.network;
+
+import java.io.File;
+
+import androidx.annotation.NonNull;
+
+/**
+ * Interface for providing the custom cache directory where animations downloaded via url are saved
```

Comment:
Add a period at the end of your doc and `@see Lottie#initialize` so people know what to do with this once they have implemented it.

Reply:
done